src/Pulsar.Client/Internal/ConsumerImpl.fs

```diff
@@ -145,7 +154,7 @@ type internal ConsumerImpl internal (consumerConfig: ConsumerConfiguration, clie
                 Log.Logger.LogDebug("{0} update PrevBatchCumulativelyAcked", prefix)
                 batchAcker.PrevBatchCumulativelyAcked <- true
             | _ ->
-                ()
+                interceptors.OnAcknowledge(this, msgId, null)
```

Comment:
Can you please fix by moving it out of match statement and splitting into OnAcknowledge and OnAcknowledgeCumulative based on ack type?

Reply:
Completed